modules/client-configuration/pages/cobbler.adoc

```diff
@@ -82,10 +88,17 @@ This example:
 * Creates a class called ``PXE``.
 * Identifies systems as ``PXEClient`` if they are configured with PXE as the first boot priority.
 * Directs PXEClients to the Cobbler server at ``192.168.2.1``.
-* Retrieves the [path]``pxelinux.0``bootloader file.
+* Retrieves the [path]``grub/grub.efi`` bootloader file for EFI PXE clients.
+* Retrieves the [path]``pxelinux.0`` bootloader file for legacy BIOS PXE clients.
 +
 . Save the file.
 
+.Procedure: Example for configuring ISC DHCP Server using DHCPd formula
```

Comment:
```suggestion
.Procedure: Configuring an ISC DHCP Server with the DHCPd formula
```

Reply:
I am not sure if this will not confuse people. There are two examples - manual configuring and this one, using dhcpd formula - and its up to user which one, but only one, they will use.

Reply:
In that case, we should probably include some explanation to that effect in a para before the procedure.

Reply:
I added some explaining text above.